src/components/common/Layout/components/NavMenu/NavMenu.tsx

```diff
@@ -102,7 +102,7 @@ const NavMenu = (): JSX.Element => {
             <NavGroup title="Other">
               <NavButton
                 target="_blank"
-                href="https://discord.gg/KUkghUdk2G"
+                href="https://discord.gg/guildxyz"
```

Comment:
Nope, please revert it, we don't have a pretty invite anymore & this redirects to a random server, haha

Reply:
Shouldn't we remove it then?